bridge/client/app/_components/ktb-secrets-list/ktb-secrets-list.component.html

```diff
@@ -16,7 +16,7 @@ <h2>Secrets</h2>
       <dt-table [dataSource]="tableEntries">
         <ng-container dtColumnDef="secretname" dtColumnAlign="text" dtColumnProportion="8">
           <dt-header-cell *dtHeaderCellDef>Name</dt-header-cell>
-          <dt-cell *dtCellDef="let row" [textContent]="row.name || 'n/a'"></dt-cell>
+          <dt-cell *dtCellDef="let row" [textContent]="toSecret(row).name || 'n/a'"></dt-cell>
```

Comment:
why are those "typecast" methods even needed? there are no types in the html anyways right?

Reply:
We have types in HTML, but the row of a DT-Table is default `any`. That's why we adjust the type.